src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -174,6 +175,7 @@ public FeedbackSessionAttributes updateFeedbackSession(FeedbackSessionAttributes
 
         FeedbackSessionAttributes[] newAttributesFinal = new FeedbackSessionAttributes[] { null };
```

Comment:
Why do we use array here? I only see the usage of first element `newAttributesFinal [0]`

Reply:
It's a workaround in Java. Variable used in lambda should be final or semi-final.